manual/pom.xml

```diff
@@ -73,6 +73,20 @@
                     <content>${project.build.directory}/generated-docs</content>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>org.apache.felix</groupId>
+                <artifactId>maven-bundle-plugin</artifactId>
+                <version>3.2.0</version>
+                <configuration>
```

Comment:
I would define `<extension>true</extension>` and `<inherited>true</inherited>` even if it's redundant: like this, it's clear.